src/components/element-editor/prefill/prefill.tsx

```diff
@@ -41,8 +41,10 @@ export const Prefill = () => {
                 });
                 return !!validationResult ? false : true;
             } catch (error) {
-                return error.errors.some((errorMessage: string | string[]) =>
-                    errorMessage.includes("required")
+                return error.errors.some(
+                    (errorMessage: string | string[]) =>
+                        errorMessage.includes("required") ||
+                        errorMessage.includes("Invalid")
                 );
```

Comment:
on second look, actually why do we need to check the content of the error message? shouldnt it be deemed erroneous as long as validation failed?

```suggestion
               return Yup.ValidationError.isError(error);
```

Reply:
fixed in the last push